src/main/java/org/folio/circulation/domain/CreateRequestService.java

```diff
@@ -102,43 +108,37 @@ private static RequestAndRelatedRecords setRequestQueuePosition(
   }
 
   private static ResponseWritableResult<RequestAndRelatedRecords> failureDisallowedForRequestType(
-    RequestType requestType) {
+      RequestType requestType) {
 
     final String requestTypeName = requestType.getValue();
 
-    return failedValidation(format(
-      "%s requests are not allowed for this patron and item combination", requestTypeName),
-      REQUEST_TYPE, requestTypeName);
+    return failedValidation(format("%s requests are not allowed for this patron and item combination", requestTypeName),
+        REQUEST_TYPE, requestTypeName);
   }
 
   private static Result<RequestAndRelatedRecords> refuseWhenInvalidUserAndPatronGroup(
-    RequestAndRelatedRecords requestAndRelatedRecords) {
+      RequestAndRelatedRecords requestAndRelatedRecords) {
 
     Request request = requestAndRelatedRecords.getRequest();
     User requester = request.getRequester();
 
-    //TODO: Investigate whether the parameter used here is correct
-    //Should it be the userId for both of these failures?
+    // TODO: Investigate whether the parameter used here is correct
+    // Should it be the userId for both of these failures?
     if (requester == null) {
-      return failedValidation(
-        "A valid user and patron group are required. User is null",
-        "userId", null);
+      return failedValidation("A valid user and patron group are required. User is null", "userId", null);
 
     } else if (requester.getPatronGroupId() == null) {
-      return failedValidation(
-        "A valid patron group is required. PatronGroup ID is null",
-        "PatronGroupId", null);
+      return failedValidation("A valid patron group is required. PatronGroup ID is null", "PatronGroupId", null);
     } else {
       return succeeded(requestAndRelatedRecords);
     }
   }
 
-  public static Result<RequestAndRelatedRecords> refuseWhenUserHasAlreadyRequestedItem(RequestAndRelatedRecords request) {
+  public static Result<RequestAndRelatedRecords> refuseWhenUserHasAlreadyRequestedItem(
+      RequestAndRelatedRecords request) {
 
-    Optional<Request> requestOptional = request.getRequestQueue().getRequests()
-      .stream()
-      .filter(it -> isTheSameRequester(request, it) && it.isOpen())
-      .findFirst();
+    Optional<Request> requestOptional = request.getRequestQueue().getRequests().stream()
+        .filter(it -> isTheSameRequester(request, it) && it.isOpen()).findFirst();
```

Comment:
Personally, I find having the stream methods on separate lines improves readability. Do you find that the reduced number of lines is more readable?

Reply:
I don't mind splitting them up.